src/main/java/com/fazlyev/bookcrudproject/controller/AuthorController.java

```diff
@@ -0,0 +1,30 @@
+package com.fazlyev.bookcrudproject.controller;
+
+import com.fazlyev.bookcrudproject.domain.Author;
+import com.fazlyev.bookcrudproject.service.AuthorService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class AuthorController {
+    private final AuthorService authorService;
+
+    public AuthorController(AuthorService authorService) {
+        this.authorService = authorService;
+    }
+
+    @GetMapping("/bookshelf/api/authors/{name}")
+    public ResponseEntity<List<Author>> findByName(@PathVariable String name){
+        return ResponseEntity.status(HttpStatus.OK).body(authorService.findByName(name));
```

Comment:
should be covered by positive and negative tests. In current case , if you don't find the name, you'll return the 200, but I'm not sure, that it's the right response.